src/components/ModelSteps/Contact.jsx

```diff
@@ -135,14 +137,35 @@ const Contact = () => {
 
 const ContactWrapper = ({ currentStep }) => {
   const { t } = useI18n()
+  const client = useClient()
   const { illustration, text } = currentStep
-  const { formSubmit } = useFormData()
+  const { formSubmit, formData } = useFormData()
   const [onLoad, setOnLoad] = useState(false)
+  const [confirmReplaceFileModal, setConfirmReplaceFileModal] = useState(false)
+
+  const cozyFiles = formData.data.filter(d => d.file.constructor === Blob)
 
-  const submit = () => {
+  const closeConfirmReplaceFileModal = () => setConfirmReplaceFileModal(false)
+  const openConfirmReplaceFileModal = () => setConfirmReplaceFileModal(true)
+
+  const submit = useCallback(() => {
     setOnLoad(true)
     formSubmit()
-  }
+  }, [formSubmit])
+
+  const onClickReplace = useCallback(
+    isReplace => {
+      ;(async () => {
+        if (isReplace) {
+          for (const { file } of cozyFiles) {
+            await client.destroy({ _id: file.id, _type: FILES_DOCTYPE })
```

Comment:
don't we handle error case?

Reply:
Nope, this case is not in the specs and not blocking